cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -1045,7 +1046,23 @@
        ["#\"(?i)foo\""]))
   (is (= "#\"a\nb\""
          (reformat-string "#\"a\nb\""))
-      "regular expression with embedded newline"))
+      "regular expression with embedded newline")
+  (is (reformats-to? 
+       ["##Inf ##-Inf ##NaN"] 
+       ["##Inf ##-Inf ##NaN"])))
+
+(deftest test-namespaced-keywords
+  (is (reformats-to? ["(ns myns.core)"
+                      ""
+                      ""
+                      "{::foo-bar (do (map)"
```

Comment:
I wonder if it's worth adding test coverage for the following related constructs...?

* current-ns qualified map: `#::{:b 2}`
* ns-alias qualified map: `#::alias{:c 3}`

They might as well deserve a separate PR ([related issue](https://github.com/weavejester/cljfmt/issues/176)), but my concern here would be that upgrading rewrite-clj could break something and was previously not broken.

Reply:
@vemv I think this is a really good feedback, thanks.
I will add some more tests in as part of this PR today.
From my notes, [rewrite-cljs and rewrite-clj v0 could not even parse current namespace autoresolve namespaced maps ](https://github.com/clj-commons/rewrite-clj/blob/main/CHANGELOG.adoc#v10572-alpha-1)(`::{:b 2}`), so I'll try the tests first against cljfmt master.

Reply:
Tests added